### PR TITLE
Tiling

### PR DESCRIPTION
**Pull request by bellert**

### CODE REVIEW

File: dwave_micro_client_dimod/tiling.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+import dimod
+import dwave_networkx as dnx
+import dwave_embedding_utilities as embutil
+
+__all__ = ['TilingComposite']
+
+class TilingComposite(dimod.TemplateComposite):
+    """Composite to tile a small problem across a structured sampler.
+
+    Args:
```
Comment (arcondello) [inline dwave_micro_client_dimod/tiling.py]: It would be good to describe here in general what Tiling is. I think a sentence or two would suffice.

Comment (arcondello) [reply dwave_micro_client_dimod/tiling.py]: Also that this only makes sense for Chimera-structured samplers

File: dwave_micro_client_dimod/tiling.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                    # Pull those cells out into an embedding.
+                    embedding = {}
+                    for sub_i in range(sub_m):
+                        for sub_j in range(sub_n):
+                            cells[i + sub_i][j + sub_j] = False  # Mark cell as matched
+                            for u in range(2):
+                                for k in range(t):
+                                    embedding[sub_c2i[sub_i, sub_j, u, k]] = {c2i[(i + sub_i, j + sub_j, u, k)]}
+
+                    self.embeddings.append(embedding)
```
Comment (arcondello) [inline dwave_micro_client_dimod/tiling.py]: If I give a sampler without Chimera-structure, what is the exception that is raised?

File: dwave_micro_client_dimod/tiling.py (hunk 1 above)
Comment (bellert) [reply dwave_micro_client_dimod/tiling.py]: Good call. Done.

File: dwave_micro_client_dimod/tiling.py (hunk 2 above)
Comment (bellert) [reply dwave_micro_client_dimod/tiling.py]: Now throws an error when 0 tiled embeddings are found

**Pull request merged**